src/org/labkey/trialshare/TrialShareController.java

```diff
@@ -1819,7 +1819,7 @@ public NavTree appendNavTrail(NavTree root)
                 }
                 catch (IllegalArgumentException ignore) {} // Don't throw because of bad user input
             }
-            return root;
+            return (null == getPageConfig().getTitle() ? root.addChild(StringUtils.capitalize(getMode()) + " Data") : root);
         }
```

Comment:
I don't think the page title would ever be non-null at this line of code, you probably want to check whether the root has any children because addChild doesn't set the page title directly

Reply:
Good point, &  done!